server/backend/testspg/constants.py

```diff
@@ -42,40 +49,109 @@
 }
 TEST_NOT_EXIST_USER_TYPE = "NS"
 
+# Admin role level at token
+FOM_DEV_ADMIN_ROLE = "FOM_DEV_ADMIN"
+FOM_TEST_ADMIN_ROLE = "FOM_TEST_ADMIN"
+
 # ------------------- Test grant/remove access -------------------------- #
-# note:
-# test idir and bceid username might need change to a
-# real one after we enable the verfication, same for
-# forest client number
-TEST_USER_ROLE_ASSIGNMENT_FOM_DEV_CONCRETE = {
+# Note:
+# The test idir and bceid username might need change to a real one after we
+#   enable the verfication, same for forest client number.
+#
+# Please refer to flyway/local_sql/V1001__add_delegated_admin_for_testing
+#   script for pre-test admin testers setup (needed for tests)".
+#   "TEST-3-LOAD-CHILD-1" and "PTOLLEST" are being setup as delegated admin.
+#
+# Note:
+# Do not change bellow various configuration for user role assignments (ACCESS_
+#   GRANT) unless needed (refactoring is required if it is changed).
+# If there is a need for additional configuration, perhaps, better to "copy"
+#   on of them and assign individual values in specific test cases.
+#
+# Note: (variable uses these to shorter the name)
+#   _AR_ : Abstract Role
+#   _CR_ : Concrete Role
+#   _L3T_: LOAD_3_TEST
+#   _L4T_: LOAD_4_TEST
+
+ACCESS_GRANT_FOM_DEV_CR_IDIR = {
     "user_name": "fom_user_test",
-    "user_type_code": "I",
-    "role_id": TEST_FOM_DEV_REVIEWER_ROLE_ID,
+    "user_type_code": fam_constants.UserType.IDIR,
+    "role_id": FOM_DEV_REVIEWER_ROLE_ID,
 }
-TEST_USER_ROLE_ASSIGNMENT_FOM_DEV_ABSTRACT = {
+
+ACCESS_GRANT_FOM_DEV_AR_00000001_IDIR = {
     "user_name": "fom_user_test",
-    "user_type_code": fam_constants.UserType.BCEID,
-    "role_id": TEST_FOM_DEV_SUBMITTER_ROLE_ID,
-    "forest_client_number": CLIENT_NUMBER_EXISTS_ACTIVE,
+    "user_type_code": fam_constants.UserType.IDIR,
+    "role_id": FOM_DEV_SUBMITTER_ROLE_ID,
+    "forest_client_number": FC_NUMBER_EXISTS_ACTIVE_00000001,
 }
-TEST_USER_ROLE_ASSIGNMENT_FOM_TEST_CONCRETE = {
+
+ACCESS_GRANT_FOM_DEV_AR_00001018_IDIR = {
+    "user_name": "fom_user_test",
+    "user_type_code": fam_constants.UserType.IDIR,
+    "role_id": FOM_DEV_SUBMITTER_ROLE_ID,
+    "forest_client_number": FC_NUMBER_EXISTS_ACTIVE_00001018,
+}
+
+ACCESS_GRANT_FOM_TEST_CR_IDIR = {
     "user_name": "fom_user_test",
     "user_type_code": "I",
-    "role_id": TEST_FOM_TEST_REVIEWER_ROLE_ID,
+    "role_id": FOM_TEST_REVIEWER_ROLE_ID,
 }
 
-TEST_USER_ROLE_ASSIGNMENT_FOM_DEV_CONCRETE_BCEID = {
-    "user_name": "LOAD-3-TEST",
+ACCESS_GRANT_FOM_DEV_CR_BCEID_L3T = {
+    "user_name": USER_NAME_BCEID_LOAD_3_TEST,
     "user_type_code": "B",
-    "role_id": TEST_FOM_DEV_REVIEWER_ROLE_ID,
+    "role_id": FOM_DEV_REVIEWER_ROLE_ID,
 }
-TEST_USER_ROLE_ASSIGNMENT_FOM_DEV_ABSTRACT_BCEID = {
-    "user_name": "LOAD-3-TEST",
+
+ACCESS_GRANT_FOM_TEST_CR_BCEID_L3T = {
```

Comment:
I'm not sure if we actually used this one

Reply:
Ah~ I created it to be used first and decided not to use on TEST. Removed.